Content.Shared/Interfaces/GameObjects/Components/Interaction/AttackEvent.cs

```diff
@@ -51,15 +32,46 @@ public class AttackEvent : EntityEventArgs
         /// </summary>
         public IEntity? TargetEntity { get; }
 
-        public AttackEvent(IEntity user, EntityCoordinates clickLocation, bool wideAttack, EntityUid target = default)
+        /// <summary>
+        ///     Modified by the handler to indicate whether the attack succeeded.
+        /// </summary>
+        public bool Succeeded { get; set;  }
+
+        public ClickAttackEvent(IEntity user, EntityCoordinates clickLocation, EntityUid target = default)
```

Comment:
You might want to add the entity this is directed to (the one being used to attack) so this can be raised broadcasted as well